draft-birkholz-scitt-receipts.md

```diff
@@ -241,6 +274,8 @@ LeafBytes = internal_hash + HASH(internal_data) + HASH(cbor(Countersign_structur
 
 The Receipt contents structure is a CBOR array. The fields of the array in order are:
 
+> It is a map, right? 
```

Comment:
Same reasoning as for the top-level receipts structure. @henkbirkholz Feel free to chime in.